source/common/router/router_ratelimit.cc

```diff
@@ -7,6 +7,7 @@
 
 #include "envoy/config/core/v3/base.pb.h"
 #include "envoy/config/route/v3/route_components.pb.h"
+#include "envoy/extensions/filters/http/ratelimit/v3/rate_limit.pb.h"
```

Comment:
Same issue as before, depending on extension. But I think you don't need this really. What are you importing here?